.travis.yml

```diff
@@ -140,6 +140,20 @@ matrix:
   - python: 3.6
     env: TOXENV=pre-commit-pep257
     after_failure: skip
+  - stage: upload new version of python package to PYPI (only for tagged commits)
+    python: 3.6
+    deploy:
+      provider: pypi
+      server: https://upload.pypi.org/legacy/
+      on:
+        tags: true
+        all_branches: true
+        python: 3.6
```

Comment:
Just remove it? Technically, yes. But I would leave it here for safety. We could use YAML anchors to reduce version duplication.